tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -966,6 +971,20 @@ def __init__(self, logger, browser, server_config, timeout_multiplier=1,
         self.close_after_done = close_after_done
         self.cleanup_after_test = cleanup_after_test
 
+    """
+    Returns a WebDriver protocol implementation to use based on the keyword
+    arguments passed to the test harness executor.
+
+    :param require_webdriver_bidi: Whether WebDriver BiDi protocol is required.
+    :return: WebDriver protocol implementation class.
+    """
+    def protocol_cls(self, require_webdriver_bidi):
```

Comment:
Why are we passing `require_webdriver_bidi` here? The instance already has it on `self.require_webdriver_bidi`.

This is also a breaking change, because we're going from a property to a method. I'd expect this to remain a property (via `@property`).